hudi-flink/src/main/java/org/apache/hudi/streamer/HoodieFlinkStreamer.java

```diff
@@ -119,22 +123,22 @@ public static void main(String[] args) throws Exception {
         .setParallelism(conf.getInteger(FlinkOptions.WRITE_TASKS));
     if (StreamerUtil.needsAsyncCompaction(conf)) {
       pipeline.transform("compact_plan_generate",
-              TypeInformation.of(CompactionPlanEvent.class),
-              new CompactionPlanOperator(conf))
-              .uid("uid_compact_plan_generate")
-              .setParallelism(1) // plan generate must be singleton
-              .rebalance()
-              .transform("compact_task",
```

Comment:
Is your indentation correct ?

Reply:
Yes, this indentation same as HoodieTableSink.